scene/gui/graph_node.h

```diff
@@ -150,6 +152,9 @@ class GraphNode : public GraphElement {
 	void set_slot_custom_icon_left(int p_slot_index, const Ref<Texture2D> &p_custom_icon);
 	Ref<Texture2D> get_slot_custom_icon_left(int p_slot_index) const;
 
+	void set_slot_meta_left(const int p_slot_index, const Variant &p_value);
+	Variant get_slot_meta_left(const int p_slot_index) const;
```

Comment:
I think it should say "metadata" for consistency:
https://github.com/godotengine/godot/blob/c81fd6c51233a727da528cf7f74137d56b5d6efe/scene/gui/item_list.cpp#L2246-L2247
https://github.com/godotengine/godot/blob/c81fd6c51233a727da528cf7f74137d56b5d6efe/scene/gui/popup_menu.cpp#L3023
https://github.com/godotengine/godot/blob/c81fd6c51233a727da528cf7f74137d56b5d6efe/scene/gui/tree.cpp#L1822-L1823